controllers/mysqlcluster_controller.go

```diff
@@ -260,18 +290,67 @@ func generateRandomBytes(n int) ([]byte, error) {
 	return ret, nil
 }
 
+func (r *MySQLClusterReconciler) createOrUpdateConfigMap(ctx context.Context, log logr.Logger, cluster *mocov1alpha1.MySQLCluster) error {
+	cm := &corev1.ConfigMap{}
+	cm.SetNamespace(cluster.Namespace)
+	cm.SetName(uniqueName(cluster))
+
+	op, err := ctrl.CreateOrUpdate(ctx, r.Client, cm, func() error {
+		setLabels(&cm.ObjectMeta)
+		gen := mysqlConfGenerator{
+			log: log,
+		}
+		gen.mergeSection("mysqld", defaultMycnf, false)
+
+		// Set innodb_buffer_pool_size if resources.requests.memory or resources.limits.memory is specified
+		mem := getMysqldContainerRequests(cluster, corev1.ResourceMemory)
+		if mem != nil {
+			bufferSize := int64(float64(mem.Value()) * 0.7)
+			gen.mergeSection("mysqld", map[string]string{"innodb_buffer_pool_size": strconv.FormatInt(bufferSize, 10)}, false)
+		}
```

Comment:
I think it is better to round up the value in MiB.  How about this?
```suggestion
		// 128MiB is the default innodb_buffer_pool_size value
		if mem != nil && mem.Value() > (128<<20) {
			bufferSize := mem.Value() / 10 * 7
			gen.mergeSection("mysqld", map[string]string{"innodb_buffer_pool_size": fmt.Sprintf("%dM", bufferSize>>20), false)
		}
```